win32ss/user/ntuser/winpos.c

```diff
@@ -1936,6 +1937,8 @@ co_WinPosSetWindowPos(
 //      valid_rects[0].left,valid_rects[0].top,valid_rects[0].right,valid_rects[0].bottom,
 //      valid_rects[1].left,valid_rects[1].top,valid_rects[1].right,valid_rects[1].bottom);
 
+   if (WvrFlags & (WVR_HREDRAW | WVR_VREDRAW))
```

Comment:
I will approve the PR, but allow me a question:
WVR_HREDRAW I do assume stands for horizontal redraw.
WVR_VREDRAW I do assume stands for vertical redraw.
What does WVR_REDRAW (from line 2056) stand for? Is that a combination of both?

Reply:
Yes, it is a combination, the constant WVR_REDRAW is defined as (WVR_HREDRAW | WVR_VREDRAW).
According to "Winconst tool (search Windows Constants)" http://oehive.org/book/export/html/466.html

That makes me wonder whether the additional local variable could be optimized away by expanding the condition in line 2056 instead.